CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ SET(PROJECT_NAMESPACE loco-3d)
 SET(PROJECT_NAME crocoddyl)
 SET(PROJECT_DESCRIPTION "Contact RObot COntrol by Differential DYnamic programming Library (Crocoddyl)")
 SET(PROJECT_URL https://github.com/${PROJECT_NAMESPACE}/${PROJECT_NAME})
+SET(CMAKE_CXX_COMPILER /usr/bin/clang++-8)
```

Comment:
There should not be changes in CMakeLists in this PR. These are your custom settings, please remove them

Reply:
@edantec: you can just use `export CXX=/usr/bin/clang++-8` before your first `cmake ..`. And if you want to always use this one, you can put that in your `~/.bashrc` :)